packages/datadog-instrumentations/src/mquery.js

```diff
@@ -0,0 +1,96 @@
+'use strict'
+
+const {
+  channel,
+  addHook,
+  AsyncResource
+} = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+
+const startCh = channel('datadog:mquery:filter:start')
+const finishCh = channel('datadog:mquery:filter:finish')
+
+const methods = ['find', 'findOne', 'findOneAndRemove', 'count', 'distinct', 'where']
```

Comment:
Add `findOneAndDelete` which is an alias for `findOneAndRemove`

https://github.com/mongoosejs/mquery/blob/master/lib/mquery.js#L2357C52-L2357C68